src/types/Comments.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comments {
+  postId: number,
```

Comment:
Use semicolons instead of commas to separate properties in TypeScript interfaces.